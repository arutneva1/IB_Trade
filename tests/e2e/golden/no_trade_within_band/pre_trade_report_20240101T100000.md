NetLiq: 18000.00
Cash USD: 10000.00
Cash Buffer: 100.00

| symbol | target_pct | current_pct | drift_bps | price | dollar_delta | share_delta | side | est_notional | reason |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AAA | 75.00 | 33.52 | 4148.04 | 100.00 | 7466.48 | 74.6648 | BUY | 7466.48 |  |
| BBB | 25.00 | 11.17 | 1382.68 | 50.00 | 2488.83 | 49.7766 | BUY | 2488.83 |  |
| TOTAL | 100.00 | 44.69 | 5530.72 |  | 9955.31 |  |  | 9955.31 |  |
