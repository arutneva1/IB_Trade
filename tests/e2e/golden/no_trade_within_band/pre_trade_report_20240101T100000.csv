NetLiq,8500.00
Cash USD,500.00
Cash Buffer,5.00

symbol,target_pct,current_pct,drift_bps,price,dollar_delta,share_delta,side,est_notional,reason
AAA,75.0,70.63,437.02,100.0,371.47,3.7147,BUY,371.47,
BBB,25.0,23.54,145.67,50.0,123.82,2.4764,BUY,123.82,
TOTAL,100.0,94.17,582.69,,495.29,,,495.29,
